### PR TITLE
[Protobuf] Support validator transaction type in protobuf

### PR DESCRIPTION
**Pull request by larry-aptos**
## Description

* Support validator transaction at protobuf
* Also remove the unused bigquery transaction type.

More context: https://github.com/aptos-labs/aptos-core/pull/13718

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [x] Other (specify) Indexer

## How Has This Been Tested?

Lint + ut

* `dkgUpdate` proto transaction example: https://gist.github.com/larry-aptos/bfb876716dc617c5717c312d2f7c8255
* `ObservedJwkUpdate` needs devnet to verify. 

more to add

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: api/types/src/transaction.rs [hunk 1]
```diff
@@ -690,9 +697,9 @@ impl From<QuorumCertifiedUpdate> for ExportedQuorumCertifiedUpdate {
 /// A more API-friendly representation of the on-chain `aptos_types::aggregate_signature::AggregateSignature`.
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
 pub struct ExportedAggregateSignature {
-    signer_indices: Vec<usize>,
```
Comment (larry-aptos) [inline api/types/src/transaction.rs:693]: only the visibility change.

File: protos/proto/aptos/indexer/v1/raw_data.proto [hunk 2]
```diff
@@ -10,7 +10,7 @@ import "aptos/transaction/v1/transaction.proto";
 // This is for storage only.
```
Comment (larry-aptos) [inline protos/proto/aptos/indexer/v1/raw_data.proto:10]: Proto format

File: protos/typescript/src/index.aptos.ts [hunk 3]
```diff
@@ -1,6 +1,5 @@
 /* eslint-disable */
 
-export * as bigquery_schema from "./index.aptos.bigquery_schema";
```
Comment (larry-aptos) [inline protos/typescript/src/index.aptos.ts:3]: remove this; we have parquet effort going on.

**Review by rtso — COMMENTED** [review]
How do you plan to test this change?

File: protos/proto/aptos/transaction/v1/transaction.proto [hunk 4]
```diff
@@ -85,10 +86,60 @@ message GenesisTransaction {
...
+    QuorumCertifiedUpdate quorum_certified_update = 1;
+  }
+
+  message DkgUpdate {
+    message DkgTranscript {
+      uint64 epoch = 1;
+      string author = 2;
+      bytes payload = 3;
+    }
+    DkgTranscript dkg_transcript = 1;
```
[the remark this answers is not in the document]
Comment (larry-aptos) [reply protos/proto/aptos/transaction/v1/transaction.proto:139]: copied from exported dkg output. 

```
#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
pub struct ExportedDKGTranscript {
    epoch: U64,
    author: Address,
    payload: HexEncodedBytes,
}
```

**Comment by larry-aptos** [comment]
https://gist.github.com/larry-aptos/bfb876716dc617c5717c312d2f7c8255 tested locally and it looks good and can see all the events. 

example

```
{
            "key": {
              "creationNumber": "12",
              "accountAddress": "0x7da5c64f4009077729b9699f46b36fd7e3567117c50b3de6e56bc7b443c9e440"
            },
            "type": {
              "type": "MOVE_TYPES_STRUCT",
              "struct": {
                "address": "0x1",
                "module": "stake",
                "name": "DistributeRewardsEvent"
              }
            },
            "data": "{\"pool_address\":\"0x7da5c64f4009077729b9699f46b36fd7e3567117c50b3de6e56bc7b443c9e440\",\"rewards_amount\":\"0\"}",
            "typeStr": "0x1::stake::DistributeRewardsEvent"
          },
          ```

**Review by rtso — APPROVED** [review]

**Review by yuunlimm — APPROVED** [review]

**Pull request merged**